reference/cric.v2.yaml

```diff
@@ -1047,6 +1047,213 @@ paths:
         - autorizacao
       description: Recover user's password.
     parameters: []
+  /collections:
+    get:
+      summary: List Collection
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/CollectionModel'
+        '400':
```

Comment:
`/collections` should not have 400 as error. If the user make a typo with query params, we will ignore the params.